src/views/v2/event-status/single/event-statuses.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Event Status Container.
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/v2/events-status/single/event-statuses.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link    http://evnt.is/1aiy
+ *
+ * @version TBD
+ *
+ * @var \WP_Post      $event         The event post object with properties added by the `tribe_get_event` function.
+ * @var Status_Labels $status_labels An instance of the statuses handler.
+ *
+ * @see     tribe_get_event() For the format of the event object.
+ */
+namespace Tribe\Events\Event_Status;
+
+// Return if no event status.
+if ( empty( $event->event_status ) ) {
+	return;
+}
+
+?>
+<div class="tribe-common-b2 tribe-events-status-single__container">
+	<?php $this->template( "event-status/single/{$event->event_status}-status" ); ?>
+</div>
```

Comment:
This should be our block. So the class here should be `tribe-events-status-single`, going off of the `postponed-status` and `canceled-status` templates. Or maybe it should be `tribe-events-status-single-notice`, but that needs to be consistent throughout the other templates.